content/articles/getting-started-with-nuxt-3/index.md

```diff
@@ -0,0 +1,132 @@
+
+Nuxt 3 is adopting TypeScript and ES Modules (ESM)to provide an enhanced development experience. TypeScript will prevent runtime errors and enhance auto-completion.
+
+However, it's not a must to write your Nuxt 3 application in TypeScript (Using JavaScript is fine and works as expected). You'll still be able to use Nuxt with no problems.
+
+#### New file structure
+The `app.vue` file serves as the primary component of your Nuxt application. With Nuxt 3, the `pages/` directory is optional, and you can work with one page `app.vue`. Everything you add to `app.vue`, such as JavaScript and CSS, will be shared across all pages. .
+
+#### Cross-platform support
+Over the years, JavaScript has come a long way, and there are now a number of platforms that use it. Nuxt 3 will run smoothly in any JS environment. This includes Node.js, Cloudflare Workers, and Deno.
+
+### Nuxt 3 plugins
+Plugins in Nuxt allow you to extend your applications' functionalities with the JavaScript library. With Nuxt 3 plugins, you can extend the functionality of your Nuxt application in three different ways:
+ + Making the functions to be available on the client-side and server-side.
+ + Using your preferred Vue.js plugins in your Nuxt application.
+ + Using external javascript packages and modules into your Nuxt application.
+
+ You can learn more about [plugins here](https://v3.nuxtjs.org/docs/directory-structure/plugins/).
+### Why you need to use Nuxt 3
+Here are some of the main reasons Nuxt 3 is a great framework and why you should use it in your next project.
+
+1. Flexibility.
+Nuxt allows you to display your application in three different ways.
++ Pre-Rendered
+In this mode, the entire website is rendered at the build step so that you can send the fully generated HTML file to the server. The app is universal, even if each is a separate file. This means that the first page is from the server, but all other pages  in the browser are single page applications (SPAs).
++ Server Rendered
+In this mode, you can create highly dynamic sites by running a node server. Each hit is rendered to the server before the browser returns a response. Again, after the initial hit, it's still  SPA.
++ SPA Mode
+In this mode, only the bare app shell is returned when the first page loads.
+
+2. Improved performance
+Nuxt is very well optimized for web performance. Nuxt has high speed and covers most by default from preload, prefetch, and lazy-loading.
+
+3. Nuxt Bridge
+The Nuxt Bridge makes migration from Nuxt 2 to Nuxt 3 easy. This layer lets you upgrade to Nuxt 3 improvements on performance and features. Also, it makes migration to be seamless.
+
+### Conclusion
+I hope you are excited as I am about the features and possibilities Nuxt 3 provides. We learned that Nuxt 3 has numerous features that will improve the project or application's performance. Nuxt 3 is an excellent framework for making the switch.
```

Comment:
add 'as': I hope you are as excited as I am 

add 'that' here: possibilities that Nuxt 3